binder/Dockerfile

```diff
@@ -2,7 +2,7 @@
 
 # Tag will be automatically generated through pre-commit hook if any changes
 # happened in the docker/ folder
-FROM ghcr.io/projectnessie/nessie-binder-demos:1862e40ba61960ec549097259a46a2b21fb6bba0
+FROM ghcr.io/projectnessie/nessie-binder-demos:7fb962b09182c5f60ac2f80bbabb693290f3f741
```

Comment:
Weird, this shouldn't change since the `docker` folder is untouched, isn't? 🤔

Reply:
Maybe another change that caused this one. Hum.

Reply:
Opened #231 to get that one merged separately

Reply:
I suspect, all the dependabot PRs don't update that hash accordingly

Reply:
I suspect, we need something that automagically updates that hash even for dependabot PRs. Maybe some GH job for dependabot PRs - IDK